core/services/workflows/models.go

```diff
@@ -106,17 +106,42 @@ func (w *workflow) dependents(start string) ([]*step, error) {
 	return steps, nil
 }
 
-// step wraps a stepDefinition with additional context for dependencies and execution
+// step wraps a Vertex with additional context for dependencies and execution
```

Comment:
Nice!

Reply:
Vertex actually contains the `dependencies` field though, does this comment need to be modified?

Reply:
ah, yes. need to change the comment. good catch. thanks